### ISSUE TITLE
2.7.0 streams are an hour delayed

### ISSUE THREAD

**Issue by letrain02** [mrlt8/docker-wyze-bridge#1086]
I have a monitor displaying streams (webrtc) and after a while the delay is huge, over an hour behind.

**Comment by Scope666** [comment]
I have this problem also ... I notice FREQUENT connects and disconnects now, never had this problem before:

<img width="956" alt="image" src="https://github.com/mrlt8/docker-wyze-bridge/assets/31927547/011ccdac-9cef-4fe9-b4d6-b5b81c4f8ef3">

I think Wyze might have pushed recent firmware updates that are breaking things?

**Comment by letrain02** [comment]
My logs look normal. Typical bitrate doesn't match constantly. I reverted to 2.6.0. no issues.

**Comment by Scope666** [comment]
I tried 2.6.0 ... no difference on my side.  Something strange is going on.

**Comment by thehijacker** [comment]
Same issue here. RTSP goes way off the sync. Sometime even more then 5 minutes. Now on 2.6.0 and it is the same. So something changed.

**Comment by Scope666** [comment]
Yeah, I think Wyze changed a behavior or something.  There's like a pattern to the disconnects I see in the logs.  For now I'm back to the RTSP firmware, which has it's own issues but stays connected much longer.

**Comment by Haddix239** [comment]
I believe this is the same issue I'm experiencing. I haven't found a solution.
